.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: BuildAndTest
```

Comment:
Can not understand wich task is it. #3 or #5?

Reply:
This is task #4